### ISSUE TITLE
[Bug] Unjustified W0326 (mixed-task-test-settings) warning when file doesn't contain neither Test Cases nor Tasks

### ISSUE THREAD

**Issue by fialsha** [MarketSquare/robotframework-robocop#1106]
### What happened?

Robocop throws unjustified W0326 (mixed-task-test-settings) warning on following file:
```
# __init__.robot
*** Settings ***
Documentation       My task documentation.

Task Tags           common_tag_for_task_suites
```

### What command/code did you try to run?

robocop

### What is the full error message?

tasks/__init__.robot:4:1 [W] 0326 Use test-related setting 'Test Tags' if Test Cases section is used (mixed-task-test-settings)

### What did you expect to happen instead?

Task Tags should be allowed for __init__.robot files which doesn't contain neither Test Cases nor Tasks sections.

### Operating System

Linux

### Robocop version

5.3.0

**Comment by bhirsz** [comment]
It sounds like a bug I fixed 2 weeks ago but didn't release it yet (#1101). I will do the release now and you can update the thread later if the issue was fixed or not.

**Comment by bhirsz** [comment]
Released Robocop 5.4.0

**Comment by fialsha** [comment]
This issue was fixed in 5.4.0 version. Thank you!
